src/server/utils/sendSegmentEvent.ts

```diff
@@ -112,14 +114,16 @@ export const sendSegmentIdentify = async (maybeUserIds) => {
     getTraits(userIds),
     getHubspotTraits(userIds)
   ])
-  traitsArr.forEach(async (traitsWithId, idx) => {
-    const {id: userId, ...traits} = traitsWithId
+  traitsArr.forEach(async (traitsWithId) => {
+    const {id: userId, ...traits} = {
+      ...traitsWithId,
+      ...hubspotTraitsArr.find((hubspotTraitsWithId) => traitsWithId.id === hubspotTraitsWithId.id)
+    }
     const tiersCountTraits = await countTiersForUserId(userId)
     segmentIo.identify({
```

Comment:
@mattkrick do we even need this call any more? Do you remember why we're calling identify() on the client side at all?

Reply:
i believe you said in order to put stuff into hubspot it had to be through an identify call